### PR TITLE
Update VITS model card

### PR DESCRIPTION
**Pull request by princepride**
# What does this PR do?

This PR updates the VITS model card to follow the new Hugging Face model card format.

- Adds framework and optimization badges
- Updates the checkpoint discovery link to the official [[facebook?search_models=mms-tts](https://huggingface.co/facebook?search_models=mms-tts)](https://huggingface.co/facebook?search_models=mms-tts) collection
- Moves model-specific usage tips (e.g., `uroman`, seed setting) into the `> [!TIP]` section
- Includes `pipeline` and `AutoModel` code examples for easy usage

This improves the usability and discoverability of the VITS model and aligns its documentation with the newer standards used in models like Whisper.

Fixes # (no specific issue, doc enhancement)

---

## Before submitting

- [x] This PR improves the documentation
- [x] I’ve read the [[contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request)](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request)
- [x] Documentation was updated accordingly

## Who can review?

@eustlb (for speech models)  
@stevhliu (for documentation)

### CODE REVIEW

File: docs/source/en/model_doc/vits.md [hunk 1]
```diff
@@ -7,168 +7,88 @@ http://www.apache.org/licenses/LICENSE-2.0
...
+        <img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
+    </div>
+</div>
 
 # VITS
 
-<div class="flex flex-wrap space-x-1">
-<img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
-</div>
+[VITS](https://hf.co/papers/2106.06103) (**V**ariational **I**nference with adversarial learning for end-to-end **T**ext-to-**S**peech) is a powerful, end-to-end speech synthesis model. It's unique because it directly synthesizes speech from text using variational inference and adversarial learning, producing natural and expressive speech with diverse rhythms and intonations.
```
Comment (stevhliu) [inline docs/source/en/model_doc/vits.md:20]:
```suggestion
[VITS (Variational Inference with adversarial learning for end-to-end Text-to-Speech)](https://hf.co/papers/2106.06103) is a end-to-end speech synthesis model, simplifying the traditional two-stage text-to-speech (TTS) systems. It's unique because it directly synthesizes speech from text using variational inference, adversarial learning, and normalizing flows to produce natural and expressive speech with diverse rhythms and intonations.
```

File: docs/source/en/model_doc/vits.md [hunk 2]
```diff
@@ -7,168 +7,88 @@ http://www.apache.org/licenses/LICENSE-2.0
...
 
-## Overview
+You can explore all official VITS checkpoints [here](https://huggingface.co/facebook?search_models=mms-tts).
```
Comment (stevhliu) [inline docs/source/en/model_doc/vits.md:22]:
```suggestion
You can find all the original VITS checkpoints under the [AI at Meta](https://huggingface.co/facebook?search_models=mms-tts) organization.
```

File: docs/source/en/model_doc/vits.md [hunk 3]
```diff
@@ -7,168 +7,88 @@ http://www.apache.org/licenses/LICENSE-2.0
...
 
-The VITS model was proposed in [Conditional Variational Autoencoder with Adversarial Learning for End-to-End Text-to-Speech](https://arxiv.org/abs/2106.06103) by Jaehyeon Kim, Jungil Kong, Juhee Son.
+> [!TIP]
+> - VITS synthesizes speech non-deterministically. For consistent results, set a fixed random seed.
+> - For languages with non-Roman alphabets (e.g., Korean or Arabic), you might need the [`uroman`](https://github.com/isi-nlp/uroman) package.
+> - Check if your tokenizer requires uroman preprocessing by inspecting the `is_uroman` attribute.
+> - Click on the VITS models in the right sidebar for more examples and use-cases!
```
Comment (stevhliu) [inline docs/source/en/model_doc/vits.md:28]:
```suggestion
> Click on the VITS models in the right sidebar for more examples of how to apply VITS.
```

File: docs/source/en/model_doc/vits.md [hunk 4]
```diff
@@ -7,168 +7,88 @@ http://www.apache.org/licenses/LICENSE-2.0
...
 
-VITS (**V**ariational **I**nference with adversarial learning for end-to-end **T**ext-to-**S**peech) is an end-to-end 
-speech synthesis model that predicts a speech waveform conditional on an input text sequence. It is a conditional variational 
-autoencoder (VAE) comprised of a posterior encoder, decoder, and conditional prior.
+The following examples show how easy it is to generate speech from text using `Pipeline` and `AutoModel`:
```
Comment (stevhliu) [inline docs/source/en/model_doc/vits.md:30]:
```suggestion
The example below demonstrates how to generate text based on an image with [`Pipeline`] or the [`AutoModel`] class.
```

File: docs/source/en/model_doc/vits.md [hunk 5]
```diff
@@ -7,168 +7,88 @@ http://www.apache.org/licenses/LICENSE-2.0
...
+# Save as WAV file
+write("hello.wav", sampling_rate, audio_data.squeeze())
+```
 
-Both the VITS and MMS-TTS checkpoints can be used with the same API. Since the flow-based model is non-deterministic, it 
-is good practice to set a seed to ensure reproducibility of the outputs. For languages with a Roman alphabet, 
-such as English or French, the tokenizer can be used directly to pre-process the text inputs. The following code example 
-runs a forward pass using the MMS-TTS English checkpoint:
+</hfoption>
+<hfoption id="AutoModel">
```
Comment (stevhliu) [inline docs/source/en/model_doc/vits.md:60]: Can clean up a bit here with this snippet:

```py
import torch
import scipy
from IPython.display import Audio
from transformers import AutoTokenizer, VitsModel, set_seed

tokenizer = AutoTokenizer.from_pretrained("facebook/mms-tts-eng")
model = VitsModel.from_pretrained("facebook/mms-tts-eng", torch_dtype=torch.float16).to("cuda")
inputs = tokenizer("Hello, my dog is cute", return_tensors="pt").to("cuda")

set_seed(555)

with torch.no_grad():
    outputs = model(**inputs)

waveform = outputs.waveform[0]
scipy.io.wavfile.write("hello.wav", rate=model.config.sampling_rate, data=waveform)

# display in Colab notebook
Audio(waveform, rate=model.config.sampling_rate)
```

File: docs/source/en/model_doc/vits.md [hunk 6]
```diff
@@ -7,168 +7,88 @@ http://www.apache.org/licenses/LICENSE-2.0
...
-inputs = tokenizer(text=uromanized_text, return_tensors="pt")
-
-set_seed(555)  # make deterministic
-with torch.no_grad():
-   outputs = model(inputs["input_ids"])
-
-waveform = outputs.waveform[0]
-```
+</hfoption>
+</hfoptions>
```
Comment (stevhliu) [inline docs/source/en/model_doc/vits.md:91]: Add a `## Notes` section after this (you can remove my backslashes "`\`" in the code blocks, I added to escape Markdown)

```md
## Notes

- Set a seed for reproducibility because VITS synthesizes speech non-deterministically.
- For languages with non-Roman alphabets (Korean, Arabic, etc.), install the [uroman](https://github.com/isi-nlp/uroman) package to preprocess the text inputs to the Roman alphabet. You can check if the tokenizer requires uroman as shown below.

   ```py
   # pip install -U uroman
   from transformers import VitsTokenizer

   tokenizer = VitsTokenizer.from_pretrained("facebook/mms-tts-eng")
   print(tokenizer.is_uroman)
   \```

   If your language requires uroman, the tokenizer automatically applies it to the text inputs. Python >= 3.10 doesn't require any additional preprocessing steps. For Python < 3.10, follow the steps below.

   ```bash
   git clone https://github.com/isi-nlp/uroman.git
   cd uroman
   export UROMAN=$(pwd)
   \```

   Create a function to preprocess the inputs. You can either use the bash variable `UROMAN` or pass the directory path directly to the function.

   ```py
   import torch
   from transformers import VitsTokenizer, VitsModel, set_seed
   import os
   import subprocess

   tokenizer = VitsTokenizer.from_pretrained("facebook/mms-tts-kor")
   model = VitsModel.from_pretrained("facebook/mms-tts-kor")

   def uromanize(input_string, uroman_path):
       """Convert non-Roman strings to Roman using the `uroman` perl package."""
       script_path = os.path.join(uroman_path, "bin", "uroman.pl")

       command = ["perl", script_path]

       process = subprocess.Popen(command, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
       # Execute the perl command
       stdout, stderr = process.communicate(input=input_string.encode())

       if process.returncode != 0:
           raise ValueError(f"Error {process.returncode}: {stderr.decode()}")

       # Return the output as a string and skip the new-line character at the end
       return stdout.decode()[:-1]

   text = "이봐 무슨 일이야"
   uromanized_text = uromanize(text, uroman_path=os.environ["UROMAN"])

   inputs = tokenizer(text=uromanized_text, return_tensors="pt")

   set_seed(555)  # make deterministic
   with torch.no_grad():
      outputs = model(inputs["input_ids"])

   waveform = outputs.waveform[0]
   \```
```

**Review by stevhliu — COMMENTED** [review]
Really nice job! 🤗

**Comment by princepride** [comment]
@stevhliu Thank you for your suggestion. I've made the changes—please review it again 😊

File: docs/source/en/model_doc/vits.md (hunk 6 above)
Comment (stevhliu) [reply docs/source/en/model_doc/vits.md:88]: Hey, I think you may have forgotten to resolve this comment 👆 It'd be good to have a `## Notes` section with this information

Comment (princepride) [reply docs/source/en/model_doc/vits.md:82]: Thanks for the reminder! I've added the Note content.

**Comment by princepride** [comment]
@stevhliu Sorry, I have add the Note content, can you review it again  😊

**Comment by princepride** [comment]
@stevhliu Hi, can you review the update again? 😊

**Review by stevhliu — APPROVED** [review]
Great work, thanks again!
